### DISCUSSION TITLE
对Glibc的要求有点高哇

### DISCUSSION THREAD

**Discussion by HelyaHsiung** [pot-app/pot-desktop#2] [General]
Debian11的c语言库才更新到2.31，无法满足pot的依赖，不敢轻易升级c语言库🤔

**Comment by Pylogmon** [comment]
低版本可以安装pot_<version>_amd64_deepin.deb，是在deepin上编译的，glibc版本2.28,这是最低的了，再低nodejs都不支持了。

**Reply by HelyaHsiung** [reply]
OK

**Reply by Pylogmon** [reply]
看来不光deepin的glibc版本低，我改个名字叫universal吧
